drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -116,20 +120,24 @@ function DragAndDropTemplates(configuration) {
                 // so all placed items are always in the correct zone.
                 description_content = gettext('Correctly placed in: {zone_title}').replace('{zone_title}', zone_title);
             }
-            var item_description = h(
+            var item_description_id = configuration.url_name + '-item-' + item.value + '-description';
+            item_content.properties.attributes = { 'aria-describedby': item_description_id };
+            item_description = h(
                 'div',
-                { key: item.value + '-description', id: item_description_id, className: 'sr' },
+                { key: item.value + '-description', id: item_description_id, className: descriptionClassName },
                 description_content
             );
-        } else {
-            var item_description = h(
-              'div',
-              { id: item_description_id, className: 'sr'},
-              gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.')
-            );
         }
         children.splice(1, 0, item_description);
         children.splice(1, 0, item_content);
+
+        var itemSRNote = h(
```

Comment:
See [Marks' comment](https://openedx.atlassian.net/browse/TNL-6031?focusedCommentId=230658&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-230658) about having to use an `aria-live` region in addition to the SR note, although I'm not sure where that region would go.